spring-kafka/src/main/java/org/springframework/kafka/listener/DefaultAfterRollbackProcessor.java

```diff
@@ -126,7 +117,8 @@ public DefaultAfterRollbackProcessor(BiConsumer<ConsumerRecord<?, ?>, Exception>
 	public DefaultAfterRollbackProcessor(@Nullable BiConsumer<ConsumerRecord<?, ?>, Exception> recoverer,
 			int maxFailures) {
 
-		this.failureTracker = new FailedRecordTracker(recoverer, new FixedBackOff(0L, maxFailures - 1), LOGGER);
+		// Remove super CTOR when this is removed.
```

Comment:
Should we add `TODO` here do not forget to address it when time comes for fixing ?

Reply:
We could but we already have it flagged as deprecated, which is trigger enough, I think.